src/components/Calculator.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+function Calculator() {
+  return (
+    <div className="calculator">
+      <h1>Math magicians</h1>
+      <div className="screen-container">0</div>
+      <div className="touch-pad">
+        <button className="gray-color" type="button">AC</button>
+        <button className="gray-color" type="button">+/-</button>
+        <button className="gray-color" type="button">%</button>
+        <button className="orange" type="button">÷</button>
+
+        <button className="gray-color" type="button">7</button>
+        <button className="gray-color" type="button">8</button>
+        <button className="gray-color" type="button">9</button>
+        <button className="orange" type="button">x</button>
+
+        <button className="gray-color" type="button">4</button>
+        <button className="gray-color" type="button">5</button>
+        <button className="gray-color" type="button">6</button>
+        <button className="orange" type="button">-</button>
+
+        <button className="gray-color" type="button">1</button>
+        <button className="gray-color" type="button">2</button>
+        <button className="gray-color" type="button">3</button>
+        <button className="orange" type="button">+</button>
+      </div>
+      <div className="sub-touch-pad">
+        <button className="gray-color zero-btn" type="button">0</button>
+        <button className="gray-color dot-btn" type="button">.</button>
+        <button className="orange equal-btn" type="button">=</button>
```

Comment:
- [ ] You did a great job 😃 at adding the component of the calculator, but I see that you need to enhance the style it little to match the original design, so kindly : 
- Try 💯 to make the border of the buttons to be like the original Design, so please try to use the correct color of the border to make it nice like the original design 😊
- Consider increasing the padding of the first row of buttons of the calculator as you here that the button's padding is less than the original design, so let's fix that to make it like the design 🚀
- It will be great 👌if you center the component of the calculator design on the page as you see here it is consistent to the left, so it will be nice for the eye if you center it on the Page 😃👌
- Also please try to increase the height of the input as you see here it is less than the original design, so let's do that 💯

| Original design | Your design| 
| --- | --- |
|  ![image](https://user-images.githubusercontent.com/83360636/226605250-5c40a5e9-c265-47ef-b9b2-226257aef245.png)    |   ![image](https://user-images.githubusercontent.com/83360636/226605211-cd87cdc8-2e03-4c84-a6bd-ed93d7e426af.png)    |